test/postServiceTest.js

```diff
@@ -0,0 +1,22 @@
+/* global it, describe */
+const sinon = require('sinon');
+const assert = require('assert');
+const Post = require('../models/Post');
+const PostService = require('../services/PostService');
+
+describe('PostService', () => {
+  describe('getOnePost(post_id)', () => {
+    it('should return correct post', () => {
+      const stub = sinon.stub(Post, 'findById').returns('post');
+      const post = PostService.getOnePost('postId');
```

Comment:
This is really weird since this unit test basically says that `getOnePost()` accepts a String as a parameter

Reply:
changed